install/kubernetes/helm/subcharts/mixer/templates/clusterrole.yaml

```diff
@@ -18,10 +18,7 @@ rules:
 - apiGroups: [""]
   resources: ["configmaps", "endpoints", "pods", "services", "namespaces", "secrets", "replicationcontrollers"]
   verbs: ["get", "list", "watch"]
-- apiGroups: ["extensions"]
-  resources: ["replicasets"]
```

Comment:
replicasets not needed for mixer?

Reply:
@sdake 
Both the `extensions` and `apps` apiGroups contain a `replicasets` resource, so I merged this rule with the one below it (see `- apiGroups: ["extensions", "apps"]`).